RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
another, better, option is to make a temporary EcalDigiCollection out of the single itdg and then call the new method.